### PR TITLE
Fix iOS Gyroscope crash issue

### LINKED ISSUE #352: iOS Gyroscope crashes
I am trying to run Gyroscope app on iOS device but it crashes as soon as I press the enable button.
Below are the complete logs:

> 2017-07-17 21:48:56.118275+0530 gyroscope[23635:10430889] [DYMTLInitPlatform] platform initialization successful
> 2017-07-17 21:48:56.926256+0530 gyroscope[23635:10430716] [Accessibility] ****************** Loading GAX Client Bundle ****************
> 2017-07-17 21:48:57.393720+0530 gyroscope[23635:10430716] Available orientation: KIVY_ORIENTATION=LandscapeLeft LandscapeRight Portrait PortraitUpsideDown
> 2017-07-17 21:48:57.393863+0530 gyroscope[23635:10430716] PythonHome is: /var/containers/Bundle/Application/BF58B956-AA98-4435-AB2C-0B43440E9F2D/gyroscope.app
> 2017-07-17 21:48:57.394817+0530 gyroscope[23635:10430716] Initializing python
> 2017-07-17 21:48:57.508802+0530 gyroscope[23635:10430716] Running main.pyo: /var/containers/Bundle/Application/BF58B956-AA98-4435-AB2C-0B43440E9F2D/gyroscope.app/YourApp/main.pyo
> [INFO   ] [Kivy        ] v1.10.0
> [INFO   ] [Python      ] v2.7.1 (r271:86832, Jul  7 2017, 22:20:33) 
> [GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)]
> [INFO   ] [Factory     ] 194 symbols loaded
> [INFO   ] [Image       ] Providers: img_imageio, img_tex (img_dds, img_sdl2, img_ffpyplayer, img_pil, img_gif ignored)
> [INFO   ] [OSC         ] using <thread> for socket
> [INFO   ] [Window      ] Provider: sdl2
> 2017-07-17 21:48:57.980231+0530 gyroscope[23635:10430716] Metal GPU Frame Capture Enabled
> 2017-07-17 21:48:57.980588+0530 gyroscope[23635:10430716] Metal API Validation Enabled
> 2017-07-17 21:48:58.039628+0530 gyroscope[23635:10430716] libMobileGestalt MobileGestaltSupport.m:153: pid 23635 (gyroscope) does not have sandbox access for frZQaeyWLUvLjeuEK43hmg and IS NOT appropriately entitled
> 2017-07-17 21:48:58.039685+0530 gyroscope[23635:10430716] libMobileGestalt MobileGestalt.c:550: no access to InverseDeviceID (see <rdar://problem/11744455>)
> [INFO   ] [GL          ] Using the "OpenGL ES 2" graphics system
> [INFO   ] [GL          ] Backend used <sdl2>
> [INFO   ] [GL          ] OpenGL version <OpenGL ES 2.0 Metal - 39.9>
> [INFO   ] [GL          ] OpenGL vendor <Apple Inc.>
> [INFO   ] [GL          ] OpenGL renderer <Apple A9 GPU>
> [INFO   ] [GL          ] OpenGL parsed version: 2, 0
> [INFO   ] [GL          ] Shading version <OpenGL ES GLSL ES 1.00>
> [INFO   ] [GL          ] Texture max size <4096>
> [INFO   ] [GL          ] Texture max units <8>
> [INFO   ] [Window      ] auto add sdl2 input provider
> [INFO   ] [Window      ] virtual keyboard not allowed, single mode, not docked
> [INFO   ] [Text        ] Provider: sdl2
> Got dlopen error on Foundation: dlopen(/System/Library/Frameworks/Foundation.framework/Versions/Current/Foundation, 1): image not found
> Got fallback dlopen error on Foundation: dlopen(/Groups/System/Library/Frameworks/Foundation.framework/Versions/Current/Foundation, 1): image not found
> [INFO   ] [Base        ] Start application main loop
> [INFO   ] [GL          ] NPOT texture support is available
>  Traceback (most recent call last):
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/plyer/utils.py", line 89, in _ensure_obj
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/plyer/platforms/ios/gyroscope.py", line 43, in instance
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/plyer/platforms/ios/gyroscope.py", line 15, in __init__
>    File "pyobjus/pyobjus_types.pxi", line 620, in pyobjus.pyobjus.ObjcClassInstance.__getattribute__ (pyobjus/pyobjus.c:17806)
>  AttributeError: 'CMMotionManager' object has no attribute 'setGyroscopeUpdateInterval_'
> [INFO   ] [Base        ] Leaving application in progress...
>  Traceback (most recent call last):
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/main.py", line 110, in <module>
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/app.py", line 828, in run
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/base.py", line 504, in runTouchApp
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/core/window/window_sdl2.py", line 663, in mainloop
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/core/window/window_sdl2.py", line 405, in _mainloop
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/base.py", line 342, in idle
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/base.py", line 327, in dispatch_input
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/base.py", line 293, in post_dispatch_input
>    File "kivy/_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy/_event.c:8191)
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/uix/behaviors/button.py", line 179, in on_touch_up
>    File "kivy/_event.pyx", line 714, in kivy._event.EventDispatcher.dispatch (kivy/_event.c:8146)
>    File "kivy/_event.pyx", line 1225, in kivy._event.EventObservers.dispatch (kivy/_event.c:14035)
>    File "kivy/_event.pyx", line 1109, in kivy._event.EventObservers._dispatch (kivy/_event.c:12816)
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/build/kivy/i386/kivy-1.10.0/iosbuild/lib/python2.7/site-packages/kivy/lang/builder.py", line 64, in custom_callback
>    File "<string>", line 18, in <module>
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/main.py", line 86, in enable
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/plyer/facades/gyroscope.py", line 98, in enable
>    File "/Users/sumitmadhwani/Documents/git/kivy-ios/gyroscope-ios/YourApp/plyer/facades/gyroscope.py", line 115, in _enable
>  NotImplementedError
> 2017-07-17 21:49:02.009030+0530 gyroscope[23635:10430716] Application quit abnormally!
> 2017-07-17 21:49:02.055112+0530 gyroscope[23635:10430716] Leaving

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/47487675-ios-gyroscope-crashes?utm_campaign=plugin&utm_content=tracker%2F77151&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F77151&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by malverick**
fixes #352

### CODE REVIEW

**Comment by malverick** [comment]
@akshayaurora please test it and merge so that I can create a pr for gyroscope uncalibrated.

File: plyer/platforms/ios/gyroscope.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class IosGyroscope(Gyroscope):
     def __init__(self):
         super(IosGyroscope, self).__init__()
         self.bridge = autoclass('bridge').alloc().init()
-        self.bridge.motionManager.setGyroscopeUpdateInterval_(0.1)
```
Comment (akshayaurora) [inline plyer/platforms/ios/gyroscope.py]: I think it is better to figure out the version of api in which this was changed and then add a if condition...

Comment (malverick) [inline plyer/platforms/ios/gyroscope.py]: will do it :)

Comment (malverick) [inline plyer/platforms/ios/gyroscope.py]: I have done it. Is it fine now?

**Comment by akshayaurora** [comment]
@malverick thanks! 👍

**Comment by malverick** [comment]
Your welcome @akshayaurora :)

**Pull request merged**